lib/ui/painting.dart

```diff
@@ -569,7 +604,7 @@ enum BlendMode {
   /// See also:
   ///
   ///  * [modulate], which always multiplies the values.
-  ///  * [screen], which always multiplies the inverses of the values.
+  ///  * [screen], which always multiplies the inverts of the values.
```

Comment:
I think in this case, 'inverses' is correct (it's the plural of the noun 'inverse', not a verb).

Reply:
Ooh, good point. I'll check for other cases of this.